### DISCUSSION TITLE
an alternative view on recipes (table form

### DISCUSSION THREAD

**Discussion by MatthiasBTH** [TandoorRecipes/recipes#1173] [Ideas]
Hello,
I just stumbled across this project by accident. That's actually what I'm looking for. Why?

Many, many years ago I started to document "my recipes" in an alternative form. I never liked the usual methodology, especially I found it very tedious when working. I found the separation between ingredients and steps quite annoying, especially when baking.

Example:

Ingredient list:
400g flour
200g sugar

Instructions:
Take the flour and mix it with the sugar ...
(question, how many ... make a look up in the list ...)

With this methodology you always have to jump back and forth between text and ingredient list. I find this clumsy. Further, I do not want to wade through long text.

I have found the optimal shape for me. The uses a table, basically from 3 columns:

Column 1: Ingredient and quantity
Column 2: in keywords the work instruction
Column 3: comment, alternatives, special notes

I attach an example here. [example.pdf](https://github.com/TandoorRecipes/recipes/files/7788961/example.pdf)
For this I started over 10 years ago with a note management, now it's over 600 pages of PDF export. Note management was not clever, a database driven methodology would be much better. 

Would that work with Tandoor?
What is your opinion?
I, havent done a deep study of the project and their opportunities. :-)

**Comment by smilerz** [comment]
@MatthiasBTH you can update the instructions with templates that will connect to the ingredients (to accommodate scaling, future editing, etc.)  you can see [here ](https://docs.tandoor.dev/features/templating/) for more info.

Adding additional columns would be difficult to implement on small formats as the information is so dense it wouldn't look very good; the columns would get stacked on each other by the responsive display.

Note that you can also store the PDFs that you spent so much time developing natively in the app.  You can then choose (or not) to enter them into the app.

**Reply by vabene1111** [reply]
thanks smilerz for the summary, i would also add that tandoor links ingredients to steps for the exact same reason you describe. You could likely put your ingredients in the field made for that and then add a markdown table into the instructions field containing two Collums, one for the instructions one for the notes. You might even be able to get away with just a list of ingredients and a few "headers" in between them.
